Sources/BraintreePayPal/BTPayPalLineItem.swift

```diff
@@ -9,6 +9,31 @@ import Foundation
     case credit
 }
 
+/// Use this option to specify  the Upc type of the lien item.
+public enum BTPayPalLineItemUpcType: String {
```

Comment:
For Obj-C merchants this enum needs to be an `Int` type. There should be some other examples in the SDK where we do this and have an internal `stringValue` property to define the strings.